optimizely/marketing-website
# add sitemap.xml

copies from root of repo to dist

## To test
check the jenkins staging workspace that `sitemap.xml` gets copied to `dist`

addresses https://optimizely.atlassian.net/browse/MKT-1771

@ihsekat
